e2etest/multichain_test.go

```diff
@@ -64,6 +64,8 @@ func TestTwoChains(t *testing.T) {
 		require.NoError(t, svr.Stop(ctx))
 	}()
 
+	time.Sleep(time.Second)
```

Comment:
WaitUntil?

Reply:
Honestly I don't find what criteria to wait until, because I asserted bolt/db.go L506 and found the ptr is not nil.